api/controllers/Depmu_EntryController.js

```diff
@@ -0,0 +1,94 @@
+/* global Prebooking DepmuEntryPrebookingValidatorService Movement */
+'use strict';
+
+var ValidationError = require('../lib/exceptions/ValidationError');
+var moment = require('moment-timezone');
+moment.tz.setDefault("Europe/London");
+
+module.exports = {
+  _config: {
+    actions: false,
+    shortcuts: false,
+    rest: false,
+    exposedMethods: [
+      'prebooking'
+    ]
+  },
+
+  prebookingOptions: (req, res) => res.ok(DepmuEntryPrebookingValidatorService.schema),
+
+  truncatePrebookings: () =>
+    Prebooking.destroy({}),
+
+  formatPrebooking: prebooking => {
+    prebooking.cid_id = prebooking.cid_id ? parseInt(prebooking.cid_id) : null;
+    prebooking.location = prebooking.location.toLowerCase().trim();
+    prebooking.task_force = prebooking.task_force.toLowerCase().trim();
+    prebooking.timestamp = new Date(prebooking.timestamp);
+    return prebooking;
+  },
+
+  populatePrebookingWithCentreAndGender: prebooking =>
+    _.memoize(Centres.getGenderAndCentreByCIDLocation)(prebooking.location)
+      .then(result =>
+        _.merge(prebooking, result)),
+
+  filterNonEmptyPrebookings: prebooking =>
+  prebooking.centre && prebooking.gender && prebooking.task_force && prebooking.timestamp,
```

Comment:
As you said ... I could make it an expression and the indentation would be correct, however then we would need to update the eslint, as it would complain about double parenthesis.